tests/cnf/core/network/security/tests/nftables-custom-firewall-test.go

```diff
@@ -0,0 +1,460 @@
+				err = validateTCPTraffic(testPodWorker0, []string{masterPodIPv4Address},
+					interfaceNameBrEx, "", portNum8888)
+				Expect(err).ToNot(HaveOccurred(),
+					"Failed to send egress TCP traffic over port 8888 to the pod on the master node")
+
+				By("Define and create a NFTables custom rule blocking ingress TCP port 8888")
+				createMCAndWaitforMCPStable(tsparams.CustomFirewallIngressPort8888, mcNftablesName)
+
+				By("Verify ingress TCP traffic is blocked over port 8888 between the master Pod and the test" +
+					" pods on the workers")
+				err = validateTCPTraffic(masterPod, ipv4NodeAddrList, interfaceNameNet1, containerFrrName, portNum8888)
+				Expect(err).To(HaveOccurred(),
+					"Successfully sent ingress TCP traffic over port 8888 to the worker nodes")
+
+				By("Verify egress TCP traffic over port 8888 between the test Pod on the worker and the master pod")
+				err = validateTCPTraffic(testPodWorker0, []string{masterPodIPv4Address},
+					interfaceNameBrEx, "", portNum8888)
+				Expect(err).ToNot(HaveOccurred(),
+					"Failed to send egress TCP traffic over port 8888 to the pod on the master node")
+			})
+	})
+})
+
+func updateMachineConfigurationNodeDisruptionPolicy() {
+	By("should update machineconfiguration cluster")
+
+	jsonBytes := []byte(`
+	{"spec":{"nodeDisruptionPolicy":
+	  {"files": [{"actions":
+	[{"restart": {"serviceName": "nftables.service"},"type": "Restart"}],
+	"path": "/etc/sysconfig/nftables.conf"}],
+	"units":
+	[{"actions":
+	[{"reload": {"serviceName":"nftables.service"},"type": "Reload"},
+	{"type": "DaemonReload"}],"name": "nftables.service"}]}}}`)
+
+	err := APIClient.Patch(context.TODO(), &ocpoperatorv1.MachineConfiguration{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "cluster",
+		},
+	}, client.RawPatch(apimachinerytype.MergePatchType, jsonBytes))
+	Expect(err).ToNot(HaveOccurred(),
+		"Failed to update the machineconfiguration cluster file")
+}
+
+func createExternalNad(name string) {
+	By("Creating external BR-EX NetworkAttachmentDefinition")
+
+	macVlanPlugin, err := define.MasterNadPlugin(coreparams.OvnExternalBridge, "bridge", nad.IPAMStatic())
+	Expect(err).ToNot(HaveOccurred(), "Failed to define master nad plugin")
+	externalNad, err := nad.NewBuilder(APIClient, name, tsparams.TestNamespaceName).
+		WithMasterPlugin(macVlanPlugin).Create()
+	Expect(err).ToNot(HaveOccurred(), "Failed to create external NetworkAttachmentDefinition")
+	Expect(externalNad.Exists()).To(BeTrue(), "Failed to detect external NetworkAttachmentDefinition")
+}
+
+func setupRemoteMultiHopTest(
+	ipv4SecurityIPList []string,
+	hubIPv4ExternalAddresses []string,
+	ipv4NodeAddrList []string,
+	hubPodWorkerName []string,
+	containerFrrName string,
+	workerNodeList []*nodes.Builder,
+	masterNodeList []*nodes.Builder) *pod.Builder {
+	var (
+		externalMacVlanNADName = "external"
+		nadHubName             = "nad-hub"
+		masterPodIPv4Prefix    = "172.16.0.1/24"
```

Comment:
could you get as  a parameter : masterPodIPv4Address

Reply:
This has been changed to 
		hubIPv4Network           = "172.16.0.0/24"
		masterPodIPv4Address     = "172.16.0.1"
Not sure what you are asking?